app/services/device_tracking/has_device.rb

```diff
@@ -0,0 +1,7 @@
+module DeviceTracking
+  class HasDevice
```

Comment:
A bit of a nit here, but this name suggests that call would return a boolean here. What not call it something like `LookupDeviceForUser`?